evalml/tests/pipeline_tests/classification_pipeline_tests/test_classification.py

```diff
@@ -47,3 +47,9 @@ def test_pipeline_has_classes_property(logistic_regression_binary_pipeline_class
 
     pipeline.fit(X, y)
     pd.testing.assert_series_equal(pd.Series(pipeline.classes_), pd.Series(answer))
+
+
+def test_woodwork_classification_pipeline(logistic_regression_binary_pipeline_class):
```

Comment:
Did you mean to convert X and y to ww objects before passing to fit?

If not, then this test may be redundant? The other tests in this file are also doing the ww conversion in fit.

Same for `test_regression.py`